spec/lib/tasks/ingest_rake_spec.rb

```diff
@@ -8,7 +8,7 @@
     ENV['SOLR_INGEST_URL'] = Blacklight.connection_config[:url]
     expect { Rake::Task['ingest_info'].invoke }.to output(
       'target is set from environment variable SOLR_INGEST_URL=http://localhost:8983/solr/discovery-test'\
-      "\nSolr collection contains 10024 results.\n"
+      "\nSolr collection contains 10124 results.\n"
     ).to_stdout
     expect { Rake::Task['ingest'].invoke('symphony_test_set') }.to_not raise_error
```

Comment:
These two tests were added to test the use of environment variables in setting the target url for ingest into Solr.  I don't think you add anything by repeating both of them.  If you want to test that you can call the ingest target without raising_error maybe here would be the best place?
```suggestion
    expect { Rake::Task['ingest'].invoke('symphony_test_set') }.to_not raise_error
    expect { Rake::Task['ingest'].invoke('kuhl_test') }.to_not raise_error
```

Reply:
But note that there are no assertions about the state of the index after the ingest.

Reply:
Any suggestion or may be some sample?